integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDiagnosticsFailedCondition.java

```diff
@@ -0,0 +1,668 @@
+class ItDiagnosticsFailedCondition {
+
+  private static String domainNamespace = null;
+
+  final String adminServerName = "admin-server";
+  final String adminServerPodName = domainUid + "-" + adminServerName;
+  final String managedServerNameBase = "ms-";
+  final String cluster1Name = "mycluster";
+  String managedServerPodNamePrefix = domainUid + "-" + managedServerNameBase;
+  final int managedServerPort = 8001;
+  int replicaCount = 2;
+
+  private static String adminSecretName;
+  private static String encryptionSecretName;
+
+  private static final String domainUid = "diagnosticsdomain";
+  private static final String pvName = domainUid + "-pv"; // name of the persistent volume
+  private static final String pvcName = domainUid + "-pvc"; // name of the persistent volume claim
+  private static String opServiceAccount = null;
+  private static String opNamespace = null;
+
+  private static LoggingFacade logger = null;
+
+  /**
+   * Assigns unique namespaces for operator and domains.
+   * Pull WebLogic image if running tests in Kind cluster.
+   * Installs operator.
+   *
+   * @param namespaces injected by JUnit
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    logger.info("Assign a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace is null");
+    opNamespace = namespaces.get(0);
+
+    logger.info("Assign a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace is null");
+    domainNamespace = namespaces.get(1);
+
+    // set the service account name for the operator
+    opServiceAccount = opNamespace + "-sa";
+
+    // install and verify operator with REST API
+    installAndVerifyOperator(opNamespace, opServiceAccount, true, 0, domainNamespace);
+
+    // create pull secrets for WebLogic image when running in non Kind Kubernetes cluster
+    // this secret is used only for non-kind cluster
+    createSecretForBaseImages(domainNamespace);
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    adminSecretName = "weblogic-credentials";
+    createSecretWithUsernamePassword(adminSecretName, domainNamespace,
+            ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    encryptionSecretName = "encryptionsecret";
+    createSecretWithUsernamePassword(encryptionSecretName, domainNamespace,
+            "weblogicenc", "weblogicenc");
+  }
+
+  /**
+   * Test domain status condition with a bad model file.
+   * Verify the following conditions are generated after a introspector failure.
+   * type: Completed, status: true
+   * type: Available, status: false
+   * type: Failed, status: true
+   */
+  @Test
+  @DisplayName("Test domain status condition with bad model file")
+  void testBadModelFileStatus() {
+    // build a build with empty WebLogic domain
```

Comment:
change to -> // build an image with empty WebLogic domain

Reply:
done